table/block_based/full_filter_block.h

```diff
@@ -138,8 +138,8 @@ class FullFilterBlockReader
                           const Comparator* comparator) const;
 
  private:
-  bool full_length_enabled_;
-  size_t prefix_extractor_full_length_;
+  bool full_length_enabled_ = false;
+  size_t prefix_extractor_full_length_ = false;
```

Comment:
To follow the other coding practices, can we move these initializations into the constructor?